src/StaticAnalysis/CachingFileAnalyser.php

```diff
@@ -168,13 +169,14 @@ private static function cacheVersion(): string
             return self::$cacheVersion;
         }
 
-        $buffer = '';
+        $buffer = [];
 
         foreach ((new FileIteratorFacade)->getFilesAsArray(__DIR__, '.php') as $file) {
-            $buffer .= file_get_contents($file);
+            $buffer[] = $file;
+            $buffer[] = file_get_contents($file);
         }
 
-        self::$cacheVersion = (string) crc32($buffer);
+        self::$cacheVersion = md5(implode("\0", $buffer));
```

Comment:
The `implode()` function is not imported.

Reply:
fixed